.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "editor.tabSize": 2,
```

Comment:
This file & folder .vscode can be removed

Reply:
you can remove this file.